README.md

```diff
@@ -2,55 +2,24 @@
 
 # Project Name
 
-> Description the project.
+> First microverse project
 
 
```

Comment:
[Required] - Please  Pull Request should have a descriptive title telling you what changes are inside.
[Required] - Please Pull Request should have a short summary describing introduced changes in more detail.
![image](https://user-images.githubusercontent.com/74744787/158656929-4bd96321-f5ff-4fca-bc6c-3ab49de10651.png)